wordpress-mobile/WordPress-iOS
# Add view elements to WhatIsNewView

Ref #14581

Adds base view elements to the "What's New / Feature Announcements" page.

**Notes:**
**1. the table view that will contain the actual announcements, as well as the "Find out more" button is not populated yet. At the moment, you will just see a grey rectangle in place of it; the grey background will be removed in future PRs.
2. The modal size on iPad has not been customized at the moment and it's using the default (which on iOS 12 would be full screen).**

To test:
1. Go to Me -> App Settings -> What's new in WordPress
2. Verify that the title "What's New in WordPress" appears and the version number is consistent to the actual version. For reference, see design in #14581 and screenshots below. 
3. The Continue button should dismiss the modal.

iPhone | iPad
--- | ---
<img height="380" alt="whatsnewiphone" src="https://user-images.githubusercontent.com/34376330/90567588-fa966080-e16f-11ea-97c6-954b9317eb75.png"> | <img height="380" alt="whatsnewipad" src="https://user-images.githubusercontent.com/34376330/90567616-097d1300-e170-11ea-9cfa-23c54ee1a460.png">

PR submission checklist:

- [x] ~~I have considered adding unit tests where possible.~~ Not yet added.
- [x] ~~I have considered adding accessibility improvements for my changes.~~ Not yet added.
